spatialos/UnrealGDK
# Don't convert Bat files to LF line endings

**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Bat files generate strange errors when line endings are LF instead of CRLF.
This only happens if a user downloads UnrealGDK as a zip, since otherwise, on Windows, Git (usually) automatically converts all files to CRLF on checkout.
#### Tests
1. Download the zip from this commit.
1. Check that line endings for `.bat` files are CRLF
1. `Setup.bat` executes without issues.
#### Primary reviewers
@Vatyx @m-samiec
